docker-compose.yml

```diff
@@ -0,0 +1,32 @@
+version: '3.3'
+
+services:
+  lookout:
+    image: "srcd/lookout"
+    network_mode: "host"
+    command: serve $LOOKOUT_REPOSITORY
+    depends_on:
+      - bblfsh
+      - postgres
+  bblfsh:
+    image: "bblfsh/bblfshd:v2.5.0"
+    privileged: true
+    ports:
+      - "9432:9432"
+    volumes:
+      - type: volume
+        source: drivers
+        target: /var/lib/bblfshd
+    entrypoint: ["/bin/sh"]
+    command:
+    - "-c"
+    - "bblfshd & sleep 1 && bblfshctl driver install --recommended && tail -f /dev/null"
+  postgres:
+    image: "postgres:10"
```

Comment:
Why not to use `postgres:alpine` first? 
It has smaller image size and we do not plan to extend/run anything else in there so should not have a problems with native deps